pkg/apiserver/apis/packages/types.go

```diff
@@ -68,3 +73,9 @@ type PackageStatus struct {
 	ObservedGeneration int64                     `json:"observedGeneration"`
 	Conditions         []kcv1alpha1.AppCondition `json:"conditions"`
 }
+
+type ValuesSchema struct {
+	// +optional
+	// +kubebuilder:pruning:PreserveUnknownFields
+	OpenAPISchemaV3 runtime.RawExtension `json:"openAPISchemaV3,omitempty"`
```

Comment:
should we change it to OpenAPIv3 (openAPIv3)? schema seems a bit redundant.

Reply:
Changed to `openAPIv3`